### ISSUE TITLE
Data lost after stream_tweets()/stream_tweets2() finishes

### ISSUE THREAD

**Issue by ghostbeagle** [ropensci/rtweet#354]
### Problem

Hello,
I've been having an issue the past few weeks with rtweet - when I specify a timeout for streaming with stream_tweets() or stream_tweets2(), if the function runs to completion, I will receive an error that says "Error: No Such File Exists!". I unfortunately the data I've collected at that time as well. 

I was curious if anyone may have had similar issues or know what may be wrong.

### Reproduce the problem

``` r
## insert code here

```
> hk_sept_inf_5<-stream_tweets2("香港,#HongKong,hong kong",timeout =3600)
Streaming tweets for 3600 seconds...
Finished streaming tweets!
**Error: No such file exists**
```

```

### rtweet version

``` r
## copy/paste output
packageVersion("rtweet")
```
`[1] ‘0.6.9’`

### Session info

sessionInfo()
```
 > sessionInfo()
R version 3.6.1 (2019-07-05)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS Mojave 10.14.6


Matrix products: default
BLAS:   /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.6/Resources/lib/libRlapack.dylib

Random number generation:
 RNG:     Mersenne-Twister 
 Normal:  Inversion 
 Sample:  Rounding 
 
locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] rtweet_0.6.9

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.2      crayon_1.3.4    zeallot_0.1.0   R6_2.4.0       
 [5] jsonlite_1.6    backports_1.1.4 magrittr_1.5    pillar_1.4.2   
 [9] httr_1.4.1      rlang_0.4.0     curl_4.0        vctrs_0.2.0    
[13] tools_3.6.1     readr_1.3.1     hms_0.5.1       compiler_3.6.1 
[17] pkgconfig_2.0.2 askpass_1.1     openssl_1.4.1   tibble_2.1.3```




### Token

```<Token>
<oauth_endpoint>
 request:   https://api.twitter.com/oauth/request_token
 authorize: https://api.twitter.com/oauth/authenticate
 access:    https://api.twitter.com/oauth/access_token
<oauth_app> CA Le...
  key:    EVxW ...
  secret: <hidden>
<credentials> oauth_token, oauth_token_secret```

**Comment by tommyxie** [comment]
I have the same issue with rtweet 0.6.9.

**Comment by tommyxie** [comment]
Did you try to write the tweets into a json file and then parse it? I was able to do it successfully. 
`hk_sept_inf_5<-stream_tweets("香港,#HongKong,hong kong",
                              timeout = 60,
                              file_name = "hktweets.json",
                              parse = FALSE)

hktweets <- parse_stream("hktweets.json")`

**Comment by ghostbeagle** [comment]
Hey Tommyxie, 
I did, but the parses have been unsuccessful. I've had the issue mentioned by another user in https://github.com/mkearney/rtweet/issues/356. In general,  I've been having trouble with both streaming and parsing since updating rtweet a few weeks ago :/. Hoping to resolve the issues soon.

**Comment by sbw78** [comment]
I am having the same bug. When I try to write to a json rtweet delettes the file as soon as it finishes. INFURIATING!

**Comment by llrs** [comment]
This was reported on #292 and should be closed once #472 gets merged
